CMakeLists.txt

```diff
@@ -77,6 +77,15 @@ mark_as_advanced(
   AF_USE_SYSTEM_FORGE
   AF_WITH_CPUID)
 
+# mark some CUDA_* variables from
+# find_package(CUDA) as advanced
+mark_as_advanced(
+    CUDA_SDK_ROOT_DIR
+    CUDA_TOOLKIT_ROOT_DIR
+    CUDA_HOST_COMPILER
```

Comment:
Keep this visible. It is useful for distros where the default compiler is not compatible with the one CUDA requires.

Reply:
Does changing `CUDA_HOST_COMPILER` change `CMAKE_CXX_COMPILER` & `CMAKE_C_COMPILER` automatically ? If not, it should be advanced variable like `CMAKE_C**_COMPILER` variables, otherwise it only adds to the confusion.